db/warehouse/migrate/20190716200323_migrate_shs_homeless.rb

```diff
@@ -0,0 +1,15 @@
+class MigrateShsHomeless < ActiveRecord::Migration
+  def change
```

Comment:
This should not be a migration, just make a rake task we can call manually.  It may take hours to run and we don't want to prevent deployment.